README.md

```diff
@@ -25,12 +25,106 @@
+
+### View Controllers
+
+The first thing you need to do is add a _view controller_ to the storyboard. As its name suggests, a view controller manages a single _view_. By "manage", we mean that it is responsible for coordinating how the view is drawn, and detecting any user input, such as touches or swipes. We'll talk about _views_ in a minute, but for now, let's get a view controller on that canvas.
+
+Do you remember where the _object library_ is? It's the pane at the bottom of the right-hand sidebar that contains a bunch of yellow icons. It looks like this:
+
+![Object Library](.images/xcode01.png)
+
+The object library contains all the different view objects—controllers, buttons, icons, and so forth—that can appear in your application. The top icon should be marked **View Controller**. That's what you want to add to your storyboard, so go ahead and drag that onto your canvas.
+
+![Drag a new view controller onto the canvas](.images/xcode02.png)
+
+You've just added a new view controller to your storyboard! This view controller will be responsible for managing the initial view of your application—everything the user sees when they first load your application.
+
+### Document Outline
+
+Notice that pane immediately to the left of Interface Builder's canvas, and to the right of the file listing? That's the _document outline_. When you first opened `Main.storyboard`, it was empty, and simply said **No Scenes**. Notice anything different? You should! Now that you have added a view controller, you'll see **View Controller Scene** in the document outline. There are _disclosure triangles_ next to that entry that show you all the objects associated with that scene.
+
+![Document Outline](.images/xcode03.png)
+
+The first one is labeled **View Controller**. That's the object you just added to your storyboard by dragging it from the object library to the canvas. Underneath that View Controller entry is one called **View**. And that's what we'll talk about next.
+
+### Views
+
+Simply put, a view controller manages a single *view*. But what is a view? A view is an object that draws something on the screen. That could be a button, or an image, or some text. A view may even contain other views, called _subviews_, that are responsible for drawing a smaller portion of the screen. Views can also detect when they are touched, swiped, poked, or otherwise interacted with. They may respond to user interactions themselves, or they may simply pass them on to their view controller to deal with them instead.
+
+Every view is responsible for a certain area of the screen. The default view provided with a view controller manages the _entire_ screen, but a view may contain other views that are responsible for just a small section of it.
+
+In technical terms, a view is an instance of `UIView`, a class provided by the Cocoa Touch framework, so you may often hear views referred to as `UIView`s.
+
+Let's take a look at some of the stuff you can do with a view.
+
```

Comment:
This I feel is a tough thing to introduce and you did a great job. Starting off referring to `UIView` would have been trouble for the student.